### DISCUSSION TITLE
What and how does all the different projects work together?

### DISCUSSION THREAD

**Discussion by stx-hseow** [Seagate/cortx#868] [Q&A] [answered]
I know about motr, s3-server and management-portal. What about other projects, how are they "useful" in building up Cortx?

**Comment by justinzw** [comment]
Thanks @harrison-seow-seagate ! I've created an issue here: https://github.com/Seagate/cortx/issues/869 that tracks creating a document that explains all of this.

**Comment by justinzw** [comment, marked as answer]
@harrison-seow-seagate  Check out this doc https://github.com/Seagate/cortx/blob/main/doc/Components.md that @hessio put together that should help explain some of the components and how they work together.

**Comment by stx-hseow** [comment]
Hi @justinzw, the document explains them nicely. 

I have only one question regarding HA and HARE, are they closely related? Like if HA detects software failure, does it come from HARE? Thanks.

Oh, and what does HARE stands for (if there's one).

**Reply by andriytk** [reply]
HAre - HA redesigned.

Or even better - just "Hare" (the small wild jumping creature with long ears). :)
